### PR TITLE
completed dom-event-delegation.

### PR DESCRIPTION
**Pull request by david-s-song**
https://user-images.githubusercontent.com/90665895/159543227-4585dd0a-5bd3-47df-8d95-a88af9049aac.mov

### CODE REVIEW

File: dom-event-delegation/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var $taskList = document.querySelector('.task-list');
+
+$taskList.addEventListener('click', function (e) {
+  console.log('event.target:', e.target);
+  console.log('event.target.tagName:', e.target.tagName);
+  if (e.target === 'BUTTON') {
```
Comment (BrettAlbright) [inline dom-event-delegation/main.js]: you want to check the `tagName` property here on the target property

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update

**Comment by david-s-song** [comment]
https://user-images.githubusercontent.com/90665895/159603872-92896dc4-03b8-4c30-8ccc-712b0277e64d.mov

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**